profiles/arch/amd64/package.use.force

```diff
@@ -1,6 +1,14 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Theo Anderson <telans@posteo.de> (2021-03-06)
```

Comment:
No, not force. Just set them as the default via `package.use`.

Reply:
I tried that but it doesn't enable them by default:
```
telans@genbox ~ $ emerge -vp1 clang-runtime libomp libcxx libcxxabi llvm-libunwind

These are the packages that would be merged, in order:

Calculating dependencies... done!
[ebuild   R   *] sys-libs/llvm-libunwind-12.0.0_rc2::gentoo  USE="-debug -static-libs -test" ABI_X86="(64) -32* (-x32)" 0 KiB
[ebuild   R   *] sys-libs/libomp-12.0.0_rc2::gentoo  USE="-cuda -hwloc -offload -ompt -test" ABI_X86="(64) -32* (-x32)" 0 KiB
[ebuild   R   *] sys-libs/libcxxabi-12.0.0_rc2::gentoo  USE="libunwind -static-libs -test" ABI_X86="(64) -32* (-x32)" 0 KiB
[ebuild   R   *] sys-libs/libcxx-12.0.0_rc2::gentoo  USE="(libcxxabi) libunwind -static-libs -test" ABI_X86="(64) -32* (-x32)" 0 KiB
[ebuild   R   *] sys-devel/clang-runtime-12.0.0_rc2:12.0.0::gentoo  USE="compiler-rt libcxx openmp sanitize" ABI_X86="(64) -32* (-x32)" 0 KiB

Total: 5 packages (5 reinstalls), Size of downloads: 0 KiB
telans@genbox ~ $ grep -r llvm-libunwind /etc/portage/package.use/
telans@genbox ~ $ grep -r llvm-libunwind /var/db/repos/gentoo/profiles/
/var/db/repos/gentoo/profiles/arch/amd64/no-multilib/package.use.force:sys-libs/llvm-libunwind -abi_x86_32
/var/db/repos/gentoo/profiles/arch/amd64/package.use:sys-libs/llvm-libunwind abi_x86_32
```

Reply:
Show me the code™.

Reply:
```
diff --git a/profiles/arch/amd64/package.use b/profiles/arch/amd64/package.use
index d54c8e8baca..81a683e2e0b 100644
--- a/profiles/arch/amd64/package.use
+++ b/profiles/arch/amd64/package.use
@@ -1,6 +1,14 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Theo Anderson <telans@posteo.de> (2021-03-06)
+# Enable abi_x86_32 Clang libs for compatability with -unwindlib=libunwind and alike
+sys-devel/clang-runtime abi_x86_32
+sys-libs/libcxxabi abi_x86_32
+sys-libs/libcxx abi_x86_32
+sys-libs/libomp abi_x86_32
+sys-libs/llvm-libunwind abi_x86_32
+
 # Mike Gilbert <floppym@gentoo.org> (2016-11-17)
 # Enable both efi and pc by default.
 # https://bugs.gentoo.org/600110
```

Reply:
It works for me. Are you sure you've tested it correctly, in particular that you don't have local overrides?

Reply:
Yes, in fact x11-misc/nvidia-drivers isn't abi_x86_32 by default for me either (but it's also in that package.use). Maybe it's because of my systemd profile for some reason? `default/linux/amd64/17.1/systemd`

Reply:
Nevermind, it's because I'm stupid. `-abi_x86_32` in make.conf. Who the hell known why on earth I did that